3rdParty/velocypack/src/StringRef.cpp

```diff
@@ -99,9 +103,16 @@ std::size_t StringRef::find(char c) const {
   return (p - _data);
 }
   
-std::size_t StringRef::rfind(char c) const {
+std::size_t StringRef::rfind(char c, std::size_t offset) const noexcept {
+  std::size_t length;
+  if (offset == std::string::npos || offset >= _length + 1) {
```

Comment:
```suggestion
  if (offset >= _length + 1) {
```
isn't npos `-1` ? Second part should always be true then